electrumx/lib/coins.py

```diff
@@ -3340,3 +3342,7 @@ class BitcoinVaultRegTest(Coin):
     ESTIMATE_FEE = 0.00001
     RELAY_FEE = 0.00001
     DAEMON = daemon.FakeEstimateFeeDaemon
+
+
+class BTCVAlert(BitcoinVaultRegTest):
```

Comment:
Name of the coin should be BitcoinVault3Keys

Reply:
done